rsmtool/modeler.py

```diff
@@ -269,12 +261,13 @@ def create_fake_skll_learner(df_coefficients):
 
     def train_linear_regression(self, df_train, feature_columns):
         """
-        Train `LinearRegression` (formerly empWt) -
-        A simple linear regression model.
+        Train a "LinearRegression" model.
```

Comment:
```suggestion
        Train a ``LinearRegression`` model.
```

Reply:
I'd rather leave these as quoted values since they are used as string values later in the main train function.